packages/components/filters/src/filter-menu/header/header.tsx

```diff
@@ -1,5 +1,73 @@
-function Header() {
-  return <div>header</div>;
-}
+import { useState, ReactNode } from 'react';
+import IconButton from '@commercetools-uikit/icon-button';
+import Spacings from '@commercetools-uikit/spacings';
+import { SortingIcon } from '@commercetools-uikit/icons';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+
+type TOption = {
+  value: string;
+  label: string;
+};
+
+type THeaderProps = {
+  headerLabel: ReactNode;
+  operandOptions?: Array<TOption>;
+  renderOperandsInput?: boolean;
```

Comment:
I couldn't find any usage of a `operandOptions` or `renderOperandsInput` prop. Are those leftovers maybe?